msteams-platform/tabs/how-to/using-teams-client-sdk.md

```diff
@@ -1,72 +1,469 @@
 ---
-title: Building tabs and other hosted experiences with the JavaScript client SDK
+title: Building tabs and other hosted experiences with the Microsoft Teams JavaScript client SDK
 author: heath-hamilton
 ms.author: surbhigupta
-description: Overview of the Microsoft Teams JavaScript client SDK, which can help you build Teams app experiences hosted in an <iframe>. It includes basic functions, authentication namespace, and settings namespace.
+description: Overview of the Microsoft Teams JavaScript client SDK, which can help you build  app experiences hosted in an <iframe> in Teams, Office, and Outlook.
 ms.localizationpriority: high
-keywords: teams tabs group channel configurable static SDK JavaScript personal
+keywords: teams tabs group channel configurable static SDK JavaScript personal m365
 ms.topic: conceptual
 ---
 # Building tabs and other hosted experiences with the Microsoft Teams JavaScript client SDK
 
-The Microsoft Teams JavaScript client SDK can help you create hosted experiences in Teams, which means displaying your app content in an iframe.
-
-The SDK is helpful for developing apps with any of the following Teams capabilities:
+The Microsoft Teams JavaScript client SDK can help you create hosted experiences in Teams, Office, and Outlook, where your app content is hosted in an [iframe](https://developer.mozilla.org/docs/Web/HTML/Element/iframe). The SDK is helpful for developing apps with the following Teams capabilities:
 
 * [Tabs](../../tabs/what-are-tabs.md)
-* [Task modules](../../task-modules-and-cards/what-are-task-modules.md)
+* [Dialogs (Task modules)](../../task-modules-and-cards/what-are-task-modules.md)
+
+With the latest release (v.2.0), the existing Teams client SDK (`@microsoft/teams-js`, or simply `TeamsJS`) has been refactored to enable [Teams apps to run in Outlook and Office](/m365-apps/overview.md), in addition to Microsoft Teams. From a functional perspective, the latest version of TeamsJS supports all existing (v.1.12) Teams app functionality while adding the optional ability to host Teams apps in Outlook and Office.
```

Comment:
The version should probably be (v1.12.1) -- but if not, the (v.1.) part seems wrong at least. Not sure if you want to reference the absolute latest version number of v1. (1.12.1 was released to correct a breaking change that was introduced in v1.11.)

Reply:
Unless I'm wrong about the version format. It looks like a period "." always following directly after the "v"?

Reply:
Sure, that works.